src/commcare_cloud/ansible/roles/nginx/tasks/install.yml

```diff
@@ -77,7 +77,7 @@
     mode: 0400
     owner: root
     group: root
-  when: not fake_ssl_cert and nginx_combined_cert_value
+  when: not fake_ssl_cert and nginx_combined_cert_value|default(None)
```

Comment:
same

Reply:
I think you missed these 2 when removing defaults